lib/smart_answer/calculators/coronavirus_find_support_calculator.rb

```diff
@@ -14,14 +14,45 @@ class CoronavirusFindSupportCalculator
                   :have_you_been_made_unemployed,
                   :mental_health_worries
 
-    def has_results?
-      return false if need_help_with.blank?
+    RULES = {
+      feeling_unsafe: lambda { |calculator|
+        calculator.needs_help_with?("feeling_unsafe") && calculator.feel_safe != "yes"
+      },
+      paying_bills: lambda { |calculator|
+        calculator.needs_help_with?("paying_bills") && calculator.afford_rent_mortgage_bills != "no"
+      },
+      getting_food: lambda { |calculator|
```

Comment:
So I think these rules need to be for sub sections - for example "Getting food" section is made up of the following subsections:

- If you’re finding it hard to afford food
- If you’re unable to get food

The both need rules to be shown/hidden based on their respective questions.

Reply:
Thanks @theseanything 👍  - please see updated changes.